CHANGELOG.md

```diff
@@ -4,6 +4,31 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v1.0.2](https://github.com/voxpupuli/puppet-network/tree/v1.0.2) (2023-04-06)
```

Comment:
probably because you run the `release` task already we now have two entries for `v1.0.2` in the changelog. I suggest to bump metadata.json to 1.1.0 and regenerate the changelog

Reply:
Thanks, will do.

Reply:
Hmm, changelog generator still wants to put the issue being closed as 1.1.0 activity, not a 1.0.2 activity:

```
## [v1.1.0](https://github.com/voxpupuli/puppet-network/tree/v1.1.0) (2023-04-07)

[Full Changelog](https://github.com/voxpupuli/puppet-network/compare/v1.0.2...v1.1.0)

**Closed issues:**

- The 'options' property of the network\_route type doesn't do anything [\#295](https://github.com/voxpupuli/puppet-network/issues/295)

## [v1.0.2](https://github.com/voxpupuli/puppet-network/tree/v1.0.2) (2023-04-06)

[Full Changelog](https://github.com/voxpupuli/puppet-network/compare/v1.0.1...v1.0.2)

**Implemented enhancements:**

- Allow for setting 'options' in network\_route on RHEL-like OS's [\#294](https://github.com/voxpupuli/puppet-network/pull/294) ([natemccurdy](https://github.com/natemccurdy))
- Replace the IPAddress gem with the built-in IPAddr class [\#290](https://github.com/voxpupuli/puppet-network/pull/290) ([imp-](https://github.com/imp-))

**Closed issues:**

- Unable to set correct netmask for IPv6 [\#267](https://github.com/voxpupuli/puppet-network/issues/267)

**Merged pull requests:**

- puppet-lint: autofix [\#291](https://github.com/voxpupuli/puppet-network/pull/291) ([bastelfreak](https://github.com/bastelfreak))

```